lib/rspecq/reporter.rb

```diff
@@ -116,16 +116,18 @@ def humanize_duration(seconds)
     def flaky_jobs_to_sentry(jobs, build_duration)
       return if jobs.empty?
 
-      Raven.capture_message("Flaky jobs detected", level: "warning", extra: {
-        build: @build_id,
-        build_timeout: @timeout,
-        queue: @queue.inspect,
-        object: self.inspect,
-        pid: Process.pid,
-        flaky_jobs: jobs,
-        flaky_jobs_count: jobs.count,
-        build_duration: build_duration
-      })
+      jobs.each do |job|
+        Raven.capture_message("[Flaky] #{job}", level: "warning", extra: {
```

Comment:
We might want to remove the test definition (e.g. `[1:15]`) in order to group events per test case

Reply:
You mean "per spec file"? In that case, yes, I believe that's the correct way to go. The reasoning is that `foo_spec.rb[1:3]` might not always refer to the same test case, as examples/describe blocks might me added/deleted from the file.

Reply:
2a71d84